### DISCUSSION TITLE
Script does not search for 3700,3900,rx6800xt please take a look at my .env file

### DISCUSSION THREAD

**Discussion by ufomike** [jef/streetmerchant#997] [Help]
### Take a look at my .env file 

SHOW_ONLY_BRANDS="nvidia,asus,evga,microsoft,sony,amd"
SHOW_ONLY_MODELS="founders edition,rog strix,tuf,ftw3,xboxsx,ps5 console,ps5 digital,xbox series x,"
SHOW_ONLY_SERIES="3080,3700,3900,sonyps5c,sonyps5de,xboxsx,rx6800xt"

**Comment by bboy11501** [comment]
where did u find ur .env file? mine doesn't exist in my folder

**Comment by TyrantRex** [comment]
> where did u find ur .env file? mine doesn't exist in my folder

I was able to find mine with a text editor like Atom or SublimeText. Just add the streetmerchant folder as a project directory.

> ### Take a look at my .env file
> SHOW_ONLY_BRANDS="nvidia,asus,evga,microsoft,sony,amd"
> SHOW_ONLY_MODELS="founders edition,rog strix,tuf,ftw3,xboxsx,ps5 console,ps5 digital,xbox series x,"
> SHOW_ONLY_SERIES="3080,3700,3900,sonyps5c,sonyps5de,xboxsx,rx6800xt"

For the rx6800xt, try adding "amd reference" to model. IDK what's going on with the rest.

**Comment by hideouspenguinboy** [comment]
There are no models for 6800xt except "amd reference" in the scripts, even though it DOES search for the different AIBs. Though it's missing a lot of 6800/xt web addresses.

**Comment by stevezub** [comment]
> ### Take a look at my .env file
> SHOW_ONLY_BRANDS="nvidia,asus,evga,microsoft,sony,amd"
> SHOW_ONLY_MODELS="founders edition,rog strix,tuf,ftw3,xboxsx,ps5 console,ps5 digital,xbox series x,"
> SHOW_ONLY_SERIES="3080,3700,3900,sonyps5c,sonyps5de,xboxsx,rx6800xt"

Do you mean 3070 and 3090? Not 3070 and 3090.
